src/Middleware/ResourceServerMiddleware.php

```diff
@@ -44,11 +45,17 @@ class ResourceServerMiddleware
     private $resourceServer;
 
     /**
+     * @var ServerOptions
+     */
+    private $serverOptions;
+
+    /**
      * @param ResourceServerInterface $resourceServer
      */
-    public function __construct(ResourceServerInterface $resourceServer)
+    public function __construct(ResourceServerInterface $resourceServer, ServerOptions $serverOptions)
```

Comment:
yeah, I saw too this but didn't want to change it all over the place again. 

But if I do I think it is be better to simply pass the option itself, unless there are more then 3 or 4 constructor arguments or the instance needs to pass them on to other dependencies. What do you think?